### ISSUE TITLE
Imports problems after upgrading to 5.3.25

### ISSUE THREAD

**Issue by basulle** [BelledonneCommunications/linphone-android#2116]
Installed 5.3.25 version by setting
    debugImplementation "org.linphone.no-video:linphone-sdk-android-debug:5.3.25"
    releaseImplementation "org.linphone.no-video:linphone-sdk-android:5.3.25"
    
Now facing issues with imports
<img width="478" alt="image" src="https://github.com/BelledonneCommunications/linphone-android/assets/65815876/3fb3b0ce-2dd7-4c41-a179-dfb22b9d0e58">

_Originally posted by @basulle in https://github.com/BelledonneCommunications/linphone-android/issues/2115#issuecomment-1971221131_

**Comment by basulle** [comment]
Hi, @Viish 
Updated maven at the gradle file to be: 
 maven {
        name "linphone.org maven repository"
        url "https://linphone.org/maven_repository/"
        content {
            includeGroup "org.linphone"
        }
    }


Now getting an issue in android studio: Failed to resolve: org.linphone.no-video:linphone-sdk-android-debug:5.3.25

**Comment by Viish** [comment]
Hi @basulle,

The no-video debug SDK does exists: https://download.linphone.org/maven_repository/org/linphone/no-video/linphone-sdk-android-debug/5.3.25/linphone-sdk-android-debug-5.3.25.aar
Do you have the same issue if you try to build with `implementation 'org.linphone:linphone-sdk-android:5.3.25'` instead?

Cheers,

**Comment by basulle** [comment]
@Viish yes, the same issue with implementation instead of release and debug

**Comment by Viish** [comment]
And if you use `implementation 'org.linphone:linphone-sdk-android:5.4.+ ` instead?

**Comment by basulle** [comment]
Yes, the same issue here.
I can see the package at Project structure -> Dependencies, but seems like its not linked, I assume.

When I upgraded from 5.1 to 5.2.112 I was not facing this issue and everything was good.

**Comment by Viish** [comment]
Sorry but I don't see why it wouldn't work with 5.3 or 5.4 release but it does work for 5.2.
I guess there is something specific in your project configuration causing this, but I fail to see what.

**Comment by basulle** [comment]
Hi, @Viish 
Seems like we can close the issue.
In my project we are using an android project as a plugin to be installed to a react-native application. When I changed the linphone version at implementation of build gradle of the plugin its self, linphone 5.3.25 installed successfully, but...
I assume linphone is required Java 17, so I need to update a lot of stuff.

fyi, after upgading the library, it was imported correctly without includeGroug, idk if that going to cause problems at the future.

Thank you for your help!

**Comment by Viish** [comment]
Indeed SDK 5.3 is built using Java 17. I'm glad you figured it out.
